### PR TITLE
Fix broken package index

### LINKED ISSUE #1: Can't compile/upload with `arduino-cli` because `avrdudecustom` missing
This platform uses `avrdudecustom` tools (in `boards.txt` and `platform.txt`). But `tools` and `toolsDependencies` are empty in `package_amperka_index.json`. So `avrdudecustom` not installed and platform is useless:
```
Uploading compiled code to the board...
Upload tool 'avrdudecustom' not found.
Upload failed
```
Please, provide links to `avrdudecustom` distribution.
@acosinwork @Igor89 @gabbapeople

### PR DESCRIPTION
**Pull request by knopki**
Closes #1 

1. Fix windows-style line endings in package index (first commit)
2. Add latest `avrdude` from main Arduino's `package_index.json`

Someone with hardware, please test this on the clean installation! With Arduino IDE, and with `arduino-cli` only, when no package or tool installed.

P.S. Not so good name of `package_amperka_index.json`. It can be a conflict when Amperka creates another package (not Iskra).

### CODE REVIEW

**Comment by acosinwork** [comment]
LGTM

**Comment by tolikivanov** [comment]
LGTM

File: boards.txt [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+iskra-nano-pb.bootloader.tool=arduino:avrdude
 iskra-nano-pb.bootloader.low_fuses=0xFF
 iskra-nano-pb.bootloader.high_fuses=0xDE
 iskra-nano-pb.bootloader.unlock_bits=0xFF
 iskra-nano-pb.bootloader.lock_bits=0xCF
 
-iskra-nano-pb.build.mcu=atmega328p
-iskra-nano-pb.build.board=AVR_A_STAR_328PB
+iskra-nano-pb.build.mcu=atmega328pb
+iskra-nano-pb.build.board=AVR_ISKRA-NANO-PB
```
Comment (nkrkv) [inline boards.txt]: If I understand correctly this string is `#define`’d while compiling. `AVR_ISKRA-NANO-PB` is not a valid C++ identifier, so should be `AVR_ISKRA_NANO_PB`

Comment (knopki) [reply boards.txt]: Fixed. Funny thing is that `AVR_ISKRA-NANO-PB` is generated by `arduino-cli` :)

**Pull request merged**